src/agent/debuglet.ts

```diff
@@ -117,6 +118,9 @@ export class Debuglet extends EventEmitter {
   private controller_: Controller;
   private completedBreakpointMap_: {[key: string]: boolean};
 
+  private promise_: Promise<void>|null;
+  private promiseResolve_: (value?: void|PromiseLike<void>|undefined) => void;
```

Comment:
Is this the right type? When you call `this.promiseResolve_`, you don't pass any arguments in.

Reply:
This type signature was the signature for promise.resolve.
Reconsidering this I think you are right, as we do not necessary to follow the signature for promise.resolve().

I updated this signature to be () => void.